bin/argbash-init

```diff
@@ -205,132 +205,109 @@ assign_positional_args 1 "${_positionals[@]}"
 ### END OF CODE GENERATED BY Argbash (sortof) ### ])
 # [ <-- needed because of Argbash
 
-
 _variables=()
 HAVE_POSITIONAL_ARG=no
 
-
-# This should be in sync with _translit_var in stuff.m4
-_translit_var()
+# This should be in sync with _translate_var in stuff.m4
+_translate_var()
 {
-	printf "\$_arg_%s" "$1" | tr '[:upper:]' '[:lower:]' | tr '-' '_'
+	printf '${_arg_%s}' "$1" | tr '[:upper:]' '[:lower:]' | tr '-' '_'
```

Comment:
This new form is correct, but it is slightly more mind-bending than the previous one, where the dollar is explicitly escaped. Could you please expand the comment, so it is clear what this function is supposed to do?

Reply:
Ah this is cleaned up by beautify when I save.  It looks at the escape and sees it can be replaced with single quotes.